### PR TITLE
updated documentation with port-listening

### PR DESCRIPTION
**Pull request by VisheshSaluja**

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -89,3 +89,17 @@ docker run -t -i -v <YOUR_HOME_DIRECTORY>:/home/marsvegan/ -p 8070:8070 jensensu
...
+
+Geoweaver allows one to change the listening port to run multiple instances simultaneously on the same machine.
+Default port used by Geoweaver is 8070, to change the port please run the following command:
+
+```
+export GEOWEAVER_PORT='Port Number'
+```
+***Example: export GEOWEAVER_PORT=8081***
+
+Run the above command in a new terminal and then rebuild the application to start a new instance of Geoweaver running on your machine.
```
Comment (ZihengSun) [inline docs/install.md:103]: No you shouldn't `rebuild` the application. Just directly run the geoweaver using the start command. Please revise.

**Review by ZihengSun — CHANGES_REQUESTED** [review]
comments

File: docs/install.md [hunk 2]
```diff
@@ -89,3 +89,17 @@ docker run -t -i -v <YOUR_HOME_DIRECTORY>:/home/marsvegan/ -p 8070:8070 jensensu
...
+
+Geoweaver allows one to change the listening port to run multiple instances simultaneously on the same machine.
+Default port used by Geoweaver is 8070, to change the port please run the following command:
+
+```
+export GEOWEAVER_PORT='Port Number'
+```
+***Example: export GEOWEAVER_PORT=8081***
+
+Run the command above in a new terminal, then start the application(using start command) to launch a new instance of Geoweaver on your machine.
```
Comment (ZihengSun) [inline docs/install.md:103]: can you list what the start command is?

**Review by ZihengSun — CHANGES_REQUESTED** [review]
BTY

File: docs/install.md (hunk 2 above)
Comment (VisheshSaluja) [reply docs/install.md:118]: @ZihengSun please check

**Review by ZihengSun — APPROVED** [review]
LG

**Pull request merged**